src/main/java/sirius/biz/translations/MultiLanguageString.java

```diff
@@ -324,7 +324,9 @@ public boolean isWithFallback() {
     @Override
     public void setData(Map<String, String> newData) {
         // remove keys with null values first
-        super.setData(newData.entrySet()
+        super.setData(newData == null ?
+                      Collections.emptyMap() :
+                      newData.entrySet()
```

Comment:
```suggestion
        if (newData == null) {
            super.setData(Collections.emptyMap());
            return:
        }
        
        super.setData(newData.entrySet()
```
I really dont like a ternary inside of a method call :)

Reply:
Actually had it like this at first and then "optimized" it :D

But a separate if is actually cleaner I guess